types/isomorphic-git__openpgp-plugin/index.d.ts

```diff
@@ -0,0 +1,8 @@
+export const pgp: {
+    sign(options: { payload: string; secretKey: string }): Promise<{ signature: string }>;
```

Comment:
I feel like it would be worthwhile to name some of these types, e.g. `SignOptions`, `VerifyOptions`, since right now you'd need to copy the declaration to declare an input..

Reply:
sure, PR updated